### PR TITLE
fix(templates): New content item created from template will be added …

### PR DESCRIPTION
**Pull request by ioanpocol**
…on the current selected desk

### CODE REVIEW

File: client/app/scripts/superdesk-archive/module.js [hunk 1]
```diff
@@ -140,8 +140,8 @@ define([
          * @param {Object} desk when passed the item will be assigned to this desk instead of user's activeDesk.
          */
         this.addTaskToArticle = function (item, desk) {
-            if ((!item.task || !item.task.desk) && (desk || desks.activeDeskId && desks.userDesks)) {
-                var currentDesk = desk || _.find(desks.userDesks._items, {_id: desks.activeDeskId});
+            if ((!item.task || !item.task.desk) && (desk || desks.getCurrentDeskId() && desks.userDesks)) {
```
Comment (sivakuna-aap) [inline client/app/scripts/superdesk-archive/module.js]: `getCurrentDeskId()` will always return `deskId` if user is member of desk(s). This prevents from creating articles in personal workspace. That's the reason `activeDeskId` was used.

If this is been taken care of then there should be test to check if creation of article is possible or not in personal workspace.

Comment (ioanpocol) [inline client/app/scripts/superdesk-archive/module.js]: In desks dropdown is not possible to select 'Personal' anymore so the content item should be added to the current selected desk on desks dropdown.
The change should be performed for the code that adds content items to 'Personal'

Comment (sivakuna-aap) [inline client/app/scripts/superdesk-archive/module.js]: Agreed that there's no option to select personal in desks dropdown. My point was when user is either in custom workspaces or selects personal option from left then the item shouldn't get created in a desk and this is not been taken care in the changes.

**Comment by plamut** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**